barman/backup.py

```diff
@@ -589,6 +595,30 @@ def delete_backup(self, backup, skip_wal_cleanup_if_standalone=True):
 
         return True
 
+    def _set_backup_sizes(self, backup_info, fsync=False):
+        """
+        Set the actual size on disk of a backup.
+        Optionally fsync all files in the backup.
```

Comment:
```suggestion

        Optionally fsync all files in the backup.
```

Line-break between summary and "description".